packages/web/docs/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "@hive/docs",
   "type": "module",
   "private": true,
+  "imports": {
+    "#components/*": "./src/components/*"
+  },
```

Comment:
This is pretty useful when importing stuff from MDX.

I don't want to add EVERYTHING to the global `mdx-components` file with a global namespace. Especially things that won't be used that often. I think subpath patterns help. (I don't really care in TS files, because autoimport works there.)